.github/workflows/build.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -r requirements.txt
+          pip install .
```

Comment:
If we are working on deps should we add here `pip check` step? https://pip.pypa.io/en/stable/cli/pip_check/